float32 bus_voltage
float32 bus_current
float32 fet_temperature
float32 motor_temperature
uint32 active_errors
uint32 disarm_reason
float32 pos_estimate
float32 vel_estimate
float32 torque_target
float32 torque_estimate
float32 iq_setpoint
float32 iq_measured
uint32 ctrl_active_errors
uint8 axis_state
uint8 procedure_result
bool trajectory_done_flag
float32 electrical_power
float32 mechanical_power